### ISSUE TITLE
Banana Bomb Sitting Problem.

### ISSUE THREAD

**Issue by crazystorm2017** [rathena/rathena#2095]
Rathena Hash
cbc3b26

Client Date
2015-11-04 Client

Server Mode
Renewal

Issue : 
When a player got slinged by a "Banana Bomb" there is chance that a player wont be able to stand even using this /sit to stand up.

**Comment by randell1993** [comment]
Correct me if im wrong but isn't it supposed to be like that?  Players won't be able to standup for 3 seconds after being hit by banana bomb?

**Comment by crazystorm2017** [comment]
Hi I Did some test in jRO and its supposed to be 3 sec only but when I Tested it in my server the banana bomb will force you to sit at exactly 15 seconds :))

**Comment by aleos89** [comment]
Related to #688. Rytech confirmed the rates that it's at now. If it has recently changed on kRO we can adjust it!

**Comment by Rytech2** [comment]
Force Sit Success Chance= Thrower BaseLv + Thrower JobLv + Thrower DEX / 6 - Target BaseLv - Target AGI/ 4 - Target LUK / 5
Force Sit Duration In ms= 1000 * Thrower JobLv / 4

These are the formula's I extracted from aegis files back in early 2015. I found it hard to believe the duration formula and decided to test this on iRO's sakray server. The test confirmed the formula and even tho its clearly OP, I can't argue with it. Its official and must be coded in as official. Tho a setting should be added to allow server owners to set a maximum allowed duration.
